tests/modules/picard/liftovervcf/test.yml

```diff
@@ -0,0 +1,13 @@
+- name: picard liftovervcf test_picard_liftovervcf
+  command: nextflow run tests/modules/picard/liftovervcf -entry test_picard_liftovervcf -c tests/config/nextflow.config
+  tags:
+    - picard
+    - picard/liftovervcf
+  files:
+    - path: output/picard/test.lifted.vcf
+      contains:
+        - "chr22"
```

Comment:
Do you know why the VCF containing the lifted variants does not always have the same md5sum?

Reply:
The lifted.vcf.gz contains the path for the reference, which changes every time the test is run so the md5sum changes too. You can see this on the line of the lifted.vcf starting with ##reference=
It is not the case for the unlifted.vcf, so its md5sum is stable

Reply:
Ok!